docs/newbs_building_firmware.md

```diff
@@ -8,14 +8,12 @@ If you have closed and reopened your terminal window since following the first p
 
 Start by navigating to the `keymaps` folder for your keyboard.
 
-?> If you are on macOS or Windows there are commands you can use to easily open the keymaps folder.
-
-?> macOS:
+If you are on macOS or Windows there are commands you can use to easily open the keymaps folder.
 
+?> macOS:<br>
```

Comment:
The html line break here isn't needed
```suggestion
?> macOS:
```

Reply:
W/o the linebreak the command will appear in the same line  as the OS which might be confusing for a newcomer. See the attached screenshot. MacOS has no linebreak, Windows has a linebreak and TempleOS shows as it was before with two seperate boxes. I feel like the second option looks best in this case. Do you disagree with that?
![screenshot](https://user-images.githubusercontent.com/22005492/66282182-3b94c780-e873-11e9-90a4-273105fb5963.png)

Cheers

Reply:
Either way this seems to break the code formatting. I'm willing to bet Docsify does not support them in `?>` blocks, so IMO the third option with the OS as a normal heading would be best.

Reply:
How about the os either as ##### or normal text and the command wrapped in backticks (\`) so that there is no extra padding around it but it is clear that it is the command the user has to use?

Reply:
`###` and triple backticks should be clear enough, I think.

Reply:
That works for me